website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,237 @@
+| **View**           |It allows you to only view resources like app(pages, themes,app names), datasources(queries, API's) etc. With the View permission, a user can only view the existing groups and roles | App Resources/ Datasource and queries/ Groups and roles/ others. |
+| **Delete**         |It allows you to delete resources like app(pages, themes,app names), datasources(queries, API's) etc. You can also define users to have the access to delete groups and roles | App Resources/ Datasource and queries/ Groups and roles/ others. |  
+| **Export**         |Allows you to export applications in a workspace. | App Resources | 
+| **Invite**         |Allows you to invite users to a user group | Groups and Roles     |
+| **Make public**    |It allows you to make the applications public in a workspace |App Resources|
+| **Execute**        |It gives you the ability to execute queries on a datasource | Datasource and queries |
+| **Remove users**   |It gives you the ability to remove a user from a group| Groups and roles |
+
+Let's understand these permissions in detail.
+
+### App resources permissions
+
+On an application level, here are the permissions that can be assigned to user or a group -
+
+#### Create
+
+`Create` gives the ability to create applications in a workspace, pages of an application or actions (APIs, queries, JSObjects) in a page. In App resources, you can give the create permission at different hierarchical levels -
+
+##### Workspace 
+
+If you give create permission to the workspace, it gives access to create applications in the respective workspace.
+
+:::note
+Under App resources, you can only define access for the applications in a workspace. To have access for the workspace itself, go to [others](#others-permissions) tab and check the permission adjacent to workspace.
+:::
+
+:::info
+Giving `Create` permission to a workspace has a cascading effect, i.e, it gives create, edit, view, delete permission to all the applications in the workspace. 
+:::
+
+##### Application
+
+If you want the user to have create access only to a specific application in a workspace, open the respective workspace's dropdown and check the create permission adjacent to the respective application. `Create` for just an application allows you to create resources like pages, actions (APIs, queries, JSObjects). 
+
+##### Page
+
+Going granular, you can give `create` access to a page in an application. It allows you to create resources like APIs, queries, JSObjects for one page.
+
+#### Edit
+
+`Edit` allows the user to change/ modify the existing applications, pages, or actions (APIs, queries, JSObjects) in a workspace. You can edit the app name, pages, and their order, color, icon, theme etc. Edit also gives you the access for deploying an app and connecting to your Git repository. 
```

Comment:
```suggestion
**Edit** allows the user to modify the existing applications, pages, database/API queries and JS Objects in a workspace. You can edit the app name, pages, and their order, color, icon, theme etc. Edit also enables access to deploy an app and connect to a Git repository. 
```

Just a doubt, is the Edit permission the one that controls access for deploying an app and connecting to Git, or the create permission on that workspace itself

Reply:
Create permission has a cascading effect. If you check create, it gives create, edit, view, delete permissions too. Create will have all the access that relates to the other 3, Edit will give this specific access.